src/app/pages/account/groups/group-list/group-list.component.ts

```diff
@@ -154,6 +162,13 @@ export class GroupListComponent {
          }
       })
   }
+
+  showOneTimeBuiltinMsg() {
+    this.prefService.preferences.showGroupListMessage = false;
+    this.prefService.savePreferences();
+    this.dialogService.confirm(helptext.builtinMessageDialog.title, helptext.builtinMessageDialog.message, 
+      true, helptext.builtinMessageDialog.button, false, '', '', '', '', true);
+  }
```

Comment:
the dialog should not have a cancel button, and should only be closable by clicking the 'ok' button.  I believe some of these options for dialog service do that